source/guide/3.2/apis-rest-client.md

```diff
@@ -0,0 +1,324 @@
+---
+layout: bt_wiki
+title: REST Client API Reference
+category: APIs
+publish: true
+abstract: REST Client API Documentation for the Cloudify Manager
+pageord: 400
+---
+
+{%summary%}
+In this section we will talk about clients you can use to call our API, how to set them up and use them.
+
+We will elaborate on the javascript client and show how you can overcome CORS problems and use it to build your own UI.
+{%endsummary%}
+
+
+# Supported SDKs
+
+ - <a href="http://cloudify-rest-client.readthedocs.org/en/3.2/" target="_blank">python</a>
+ - <a href="https://s3.amazonaws.com/cloudifyjs.gsdev.info/3.2.0/index.html" target="_blank">javascript</a>
+
+
+{%note title=Note%}
+For using the javascript client from frontend, you will need to setup CORS manually.
+Read below to see how.
+{%endnote%}
+
+
+# Python Client
+
+To use this client run the command `pip install  cloudify-rest-client==3.2.0` or add it to your dependencies file.
```

Comment:
yes. of course. 
When 3.3 is released all of these links will be irrelevant if I use `latest`. 

On 3.3 branch or folder we should change these versions to 3.3. 

How can it work otherwise?